### DISCUSSION TITLE
How to omit date from md file YYYY-MM-DD.my-link and use date in file

### DISCUSSION THREAD

**Discussion by sts-ryan-holton** [spatie/sheets#80] [Q&A]
Hi, I've got markdown files that I'm using and I'm storing a `createdAt` variable in the YYYY-MM-DD format within the three back ticks. I'd like to omit date from the md file name. How do i achieve this?

**Comment by sebastiandedeyne** [comment]
You can configure the package to use `SlugParser` instead of `SlugWithDateParser`. Then Sheets will not try to parse a date from the filename. For the `createdAt` variable in front matter you don't need to do anything, all front matter is available in the Sheet.

In `config/sheets.php`:

```diff
 <?php
 
 return [
    'default_collection' => 'posts",

     'collections' => [

         'posts' => [
             'disk' => 'posts',
             'sheet_class' => App\Models\Post::class,
-            'path_parser' => Spatie\Sheets\PathParsers\SlugWithDateParser::class,
+            'path_parser' => Spatie\Sheets\PathParsers\SlugParser::class,
             'content_parser' => Spatie\Sheets\ContentParsers\MarkdownParser::class,
             'extension' => 'txt',
         ],
     ],
 ];
```
